src/parser/parser.ts

```diff
@@ -14,9 +19,11 @@ const pathHeading: Renderer = (path: string) => `# \`${path.replace(homedir(), '
 function wrap(contentType: string, content: string, linkPath?: string): string {
     let link = '';
     if (linkPath) {
-        link = `\n<div id="top-nav">\n<a id="top-nav-up" href="${pathToURL(linkPath)}"> ${
-            pbasename(linkPath) || '/'
-        }</a>\n</div>`;
```

Comment:
Some literal `\n`s removed here, @Tweekism pls warn me if they were necessary

My thought process is that when I get the icon alignment right, aligning the divs and such are gonna use CSS (margins/paddings) anyways

Reply:
the \n are for formatting of the actual html source, not the rendered output.

Reply:
Oh I see now. Since I added some actual linebreaks in the string it should achieve a similar neatness of the HTML I suppose. But other parts of the code don't have this goal in mind and maybe it's not important :smile: